packages/flutter/test/material/data_table_test.dart

```diff
@@ -94,9 +106,40 @@ void main() {
     ));
     await tester.pumpAndSettle(const Duration(milliseconds: 200));
 
-    await tester.tap(find.text('375'));
+    await tester.tap(find.text('375'),pointer: 2);
+    await tester.pump(const Duration(milliseconds: 100));
+    await tester.tap(find.text('375'),pointer: 3);
+
+    expect(log, <String>['cell-doubleTap: 375']);
+    log.clear();
+
+    await tester.longPress(find.text('375'),pointer: 4);
+
+    expect(log,<String>['cell-tapDown: 375' ,'cell-tapCancel: 375', 'cell-longPress: 375']);
```

Comment:
Add a comment here explaining why tapDown, and tapCancel were logged prior to longPress (might not be clear to readers unfamiliar with Flutter's gesture arena mechanics)

Reply:
Please elaborate a bit.

Reply:
Basically, it's clear to you and me that that the log is:
tap down -> tap cancel -> long press
This is because the tap down is triggered on gesture down. Then, the cancel is triggered when the gesture arena recognizes that the long press overrides the tap event so it triggers a tap cancel, followed by the long press. 

However, the series of events above might be unclear to those who have never interacted too much with the gesture arena.